packages/integrations/sitemap/src/index.ts

```diff
@@ -177,7 +177,7 @@ const createPlugin = (options?: SitemapOptions): AstroIntegration => {
 						limit: entryLimit,
 						gzip: false,
 					});
-					logger.success(`\`${OUTFILE}\` is created.`);
+					logger.success(`\`${OUTFILE}\` created at \`${fileURLToPath(dir)}`);
```

Comment:
I think we can extract `fileURLToPath(dir)` as its own variable, and maybe we can also do `path.relative(process.cwd(), destDir)` so that the logged path is shorter?

Reply:
Will do!

Reply:
@bluwy updated.